src/main/java/online/talkandtravel/service/impl/ChatServiceImpl.java

```diff
@@ -313,7 +347,8 @@ private void checkIfChatExists(User user, User companion) {
 
   private void checkIsDifferentUsers(User user, User companion) {
     if (user.getId().equals(companion.getId())) {
-      throw new IllegalArgumentException("Creation a chat with the same user. User id: %s".formatted(user.getId())) ;
+      throw new IllegalArgumentException(
```

Comment:
no. Create custom exception that extends our HttpException. Place full 'message ' into that exception class